packages/grafana-toolkit/README.md

```diff
@@ -1,55 +1,82 @@
-# Grafana Toolkit
 
-Make sure to run `yarn install` before trying anything!  Otherwise you may see unknown command grafana-toolkit and spend a while tracking that down.
+# grafana-toolkikt
+grafana-toolkit is a simple CLI that enables efficient Grafana extensions development
 
+## Why grafana-toolkit?
```

Comment:
This is an incomplete sentence (although commonly used). I would either make it a proper sentence, like "Why did we create grafana-toolkit?", or alternatively just "Rationale" or "Background".